galleries/examples/axes_grid1/demo_colorbar_with_inset_locator.py

```diff
@@ -1,15 +1,21 @@
 """
+.. _demo-colorbar-with-inset-locator:
+
 ==============================================================
 Controlling the position and size of colorbars with Inset Axes
 ==============================================================
 
-This example shows how to control the position, height, and width of
-colorbars using `~mpl_toolkits.axes_grid1.inset_locator.inset_axes`.
+This example shows how to control the position, height, and width of colorbars
+using `~mpl_toolkits.axes_grid1.inset_locator.inset_axes`.
 
 Inset axes placement is controlled as for legends: either by providing a *loc*
 option ("upper right", "best", ...), or by providing a locator with respect to
 the parent bbox.  Parameters such as *bbox_to_anchor* and *borderpad* likewise
 work in the same way, and are also demonstrated here.
+
+Users should consider using `.Axes.inset_axes` instead (see
+:ref:`colorbar_placement`).
```

Comment:
If the last two sections of the colorbar guide become one "Manually Placing Colorbars" section (see other comment), then this could link directly to that.